docs/html/topics/vcs-support.md

```diff
@@ -73,16 +73,16 @@ The supported schemes are `svn`, `svn+svn`, `svn+http`, `svn+https` and
 `svn+ssh`. Here are some of the supported forms:
 
 ```none
-MyProject @svn+https://svn.example.com/MyProject
-MyProject @svn+ssh://svn.example.com/MyProject
-MyProject @svn+ssh://user@svn.example.com/MyProject
+MyProject @ svn+https://svn.example.com/MyProject
+MyProject @ svn+ssh://svn.example.com/MyProject
+MyProject @ svn+ssh://user@svn.example.com/MyProject
 ```
 
 You can also give specific revisions to an SVN URL, like so:
 
 ```none
-MyProject @ -e svn+http://svn.example.com/svn/MyProject/trunk@2019
-MyProject @ -e svn+http://svn.example.com/svn/MyProject/trunk@{20080101}
+MyProject @ -e svn+http://svn.example.com/svn/MyProject/trunk@2019#egg=project_name
+MyProject @ -e svn+http://svn.example.com/svn/MyProject/trunk@{20080101}#egg=project_name
```

Comment:
I don't have a svn repo to test but this should be:

```suggestion
-e svn+http://svn.example.com/svn/MyProject/trunk@2019#egg=project_name
-e svn+http://svn.example.com/svn/MyProject/trunk@{20080101}#egg=project_name
```

Reply:
That looks good! Thank you!

Reply:
> I don't have a svn repo

I am having similar issues.  It'd be nice to have a list of public python svn projects to test with